stable/polaris/templates/ingress.yaml

```
@@ -17,6 +17,9 @@ metadata:
     app.kubernetes.io/instance: {{ .Release.Name }}
   name: polaris
 spec:
+{{- if and (.Values.dashboard.ingress.ingressClassName) (semverCompare ">= v1.18.0" .Capabilities.KubeVersion.Version) }}
```

It is better to stay consistent and use the same condition as at the top of the file to choose the API version:
```suggestion
{{- if and (.Values.dashboard.ingress.ingressClassName) (.Capabilities.APIVersions.Has "networking.k8s.io/v1/Ingress") }}
```